README.md

```diff
@@ -0,0 +1,9 @@
+# OpenCompose
+
+The goal of OpenCompose is to make it easier to on-board developers to Kubernetes. It is a curated higher level abstraction for specific Kubernetes resources. Very simple idea, isn't it? Developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications on Kubernetes. The developer is generally concerned with the application that is being developed.
+
+We are at a very nascent stage of this project and we have certain [ideas](https://github.com/redhat-developer/opencompose/blob/master/examples/basic.yaml) but we are definitely looking for suggestions and contributions from the Kubernetes community. Please send PRs to improve this project,
+file issues and RFEs against this repository.
+
+
+
```

Comment:
A example that shows what is current state with all supported directives will help folks know how it looks like?